README.md

```diff
@@ -327,22 +317,29 @@ the initial date.
 
 ### Full configuration file example
 
+Here are the expected environment variables:
+
+- `CRAWLERS_API_KEY`: an arbitrary api key used by the crawler to index data.
+- `GITHUB_TOKEN`: a github token for `github_organization` crawler.
+- `GITLAB_TOKEN`: a gitlab token for `gitlab_organization` crawler.
+- Optional `GERRIT_PASSWORD` for `gerrit_url` crawler.
```

Comment:
We should use GitHub, GitLab, Gerrit name when we talk about crawlers in the doc.